### DISCUSSION TITLE
X96 Max Plus Q2: Wifi (qca6174) slow/lagging - caused by wifi power management

### DISCUSSION THREAD

**Discussion by votarum** [devmfc/debian-on-amlogic#45] [General]
It appears that the issue caused by power management. I installed wireless-tools package:
`apt install wireless-tools`
and disabled wifi power management:
`/usr/sbin/iwconfig wlan0 power off`
After this lagging in ssh console stopped, and spontaneous disconnections from the access point as well gone.

**Comment by devmfc** [comment]
Serious? That is really good to hear! 

Thanks for the tip. And if it is true, I will disable CONFIG_CFG80211_DEFAULT_PS once and for all.... Maybe that wll solve it too and maybe it will solve numerous other wifi problems.

Thanks.

**Reply by votarum** [reply]
Yes, unfortunately this problem often occurs with various wifi drivers. If the device is not mobile, it is always better to disable power management.
